staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -79,8 +79,15 @@ func newCache() *cache {
 
 var spewConfig = &spew.ConfigState{DisableMethods: true, Indent: " "}
 
-func cacheKey(c *api.ExecConfig) string {
-	return spewConfig.Sprint(c)
+func cacheKey(conf *api.ExecConfig, cluster clientauthentication.Cluster) string {
+	key := struct {
+		conf    *api.ExecConfig
+		cluster clientauthentication.Cluster
+	}{
+		conf:    conf,
+		cluster: cluster,
+	}
+	return spewConfig.Sprint(key)
```

Comment:
does this do the right thing printing the runtime.Config object?

Reply:
Yeah, the new test cases in `TestCacheKey` should cover that.